server/zanata-war/src/test/java/org/zanata/dao/ProjectDAOJPATest.java

```diff
@@ -43,10 +44,10 @@ private static HProject makeProject(String slug, String name, String description
         return hProject;
     }
 
-    private void doInTransaction(Function<EntityManager, HProject> function) {
+    private void doInTransaction(Iterable<Function<EntityManager, HProject>> function) {
```

Comment:
It's just a test, but I think I would have used Runnable, not Function, since you don't care about the result.